packages/flutter/lib/src/cupertino/search_field.dart

```diff
@@ -195,6 +196,11 @@ class CupertinoSearchTextField extends StatefulWidget {
   // https://github.com/flutter/flutter/issues/13914.
   final BorderRadius? borderRadius;
 
+  /// The keyboard type for this search field.
+  ///
+  /// Defaults to TextInputType.text.
```

Comment:
Nit: Use square brackets so this gets linked in the docs: `[TextInputType.text]`.